opty/direct_collocation.py

```diff
@@ -166,6 +172,13 @@ def __init__(self, equations_of_motion, state_symbols,
             ndarrays of floats of shape(N,). Any time varying parameters in
             the equations of motion not provided in this dictionary will
             become free trajectories optimization variables.
+        known_experiment_map : dictionary, optional
```

Comment:
The `known_parameter_map` already maps numpy arrays and floats to symbols in the equations of motion. Why is a new dictionary needed?

Currently this class isn't aware of the concept of measurement data in a parameter id sense.